base/refpointer.jl

```diff
@@ -13,6 +13,11 @@ Creation of a `Ref` to a value `x` of type `T` is usually written `Ref(x)`.
 Additionally, for creating interior pointers to containers (such as Array or Ptr),
 it can be written `Ref(a, i)` for creating a reference to the `i`-th element of `a`.
 
+A `Ref` of a non-bitstype `T` referring to `undef` can be made by `Ref{T}()`.
+To check if a `Ref` refers to undef, use `isassigned(ref)`.
+For example, `isassigned(Ref{T}())` is `false` if T is a struct. This does not
+apply to primitive types. `isassigned(Ref{Int64}())` results in `true`.
+
 When passed as a `ccall` argument (either as a `Ptr` or `Ref` type), a `Ref` object will be
 converted to a native pointer to the data it references.
 
```

Comment:
This should probably be merged with the paragraph about `NULL` right below. The mention of `NULL` below is also wrong for exactly the case mentioned above...

Reply:
I removed the no invalid reference language in 2849f95